app/views/mentee_profiles/show.html.slim

```diff
@@ -3,11 +3,15 @@
     .col-md-6
       = image_tag "profile/mentee_icon.jpg", alt: "Mentee profile picture", class: "profile-icon"
     .col-md-6
-      h3.mentee-name Jane Doe
-      p From: UK
+      h3.mentee-name 
+       ='  @user.first_name 
+       ='  @user.last_name
+      p 
+       | From 
+       = @user.country
       p
-        | Currently in: Italy
-        a.left10 GMT+2
+        | Currently in:  @user.program_country
```

Comment:
Program_country and timezone need to be added in user table .. Should i add it ?